packages/web-component-tester/runner/webserver.ts

```diff
@@ -261,7 +262,7 @@ Expected to find a ${mdFilenames.join(' or ')} at: ${pathToLocalWct}/
 
     options.webserver._servers = servers.map((s) => {
       const port = s.server.address().port;
-      const hostname = s.options.hostname;
+      const hostname = s.server.address().address;
```

Comment:
Why this change?

Reply:
I thought it would make sense to bring the behaviour of how port and host were set inline with each other. 

I originally set `const port = s.options.port` but this results in a strange default of 0000 when the port isn't set. So I updated hostname to use the same logic as port instead. 

This was probably not really required. Should I remove it ?

Reply:
I'd leave it out unless we want to use a library for url construction, because if the address is an ipv6 address then we need to wrap it with square brackets. (i.e. http://[2001:4860:0:2001::68]/ is how you make a url out of the v6 address 2001:4860:0:2001::68])

Reply:
Changes made.